first_timers/first_timers.py

```diff
@@ -4,6 +4,11 @@
 import requests
 import warnings
 import re
+import json
+import os
+from github import Github
+
+git_token = os.environ.get('GIT_TOKEN')
```

Comment:
I like the idea of using a GitHub auth token.

However, instead of using the environment variable, I'd rather save the setting in a private credentials file, which is read by default here in JSON format:

https://github.com/musically-ut/first-timers-only-bot/blob/master/first_timers/run.py#L61

The GitHub access token should be saved in the file and read like the Twitter credentials are.

And if we are using the `pyGithub` package, does it offer a good API for  the search interface so that we can simplify the `requests` we make?